### DISCUSSION TITLE
Hi ~~New to here

### DISCUSSION THREAD

**Discussion by Scarlettpianpian** [community/community#64852] [New to GitHub]
Hi~My name is Sijia Hao. You can call me Scarlett. I'm from china，major in AI, a sophomore . I'm glad  to discuss with you guys  and get to know more knowledge  about AI or Internet.

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @Scarlettpianpian  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.

**Reply by Scarlettpianpian** [reply]
Oh~ ok !Thank u.
